src/main/rx/ghst.c

```diff
@@ -331,6 +336,24 @@ static bool ghstProcessFrame(const rxRuntimeState_t *rxRuntimeState)
                 }
 
             }
+        } else {
+            switch(ghstFrameType) {
+
+#if defined(USE_TELEMETRY_GHST) && defined(USE_MSP_OVER_TELEMETRY)
+                case GHST_UL_MSP_REQ:
```

Comment:
case should have the same indent as switch

Reply:
Thank you, fixed.